### ISSUE TITLE
equation render bug

### ISSUE THREAD

**Issue by yangkang2318** [transitive-bullshit/nextjs-notion-starter-kit#300]

**Comment by y1zhou** [comment]
It has to be in its own block, i.e. "display mode" of KaTeX. The `equation` environment can't be rendered inline.

**Comment by yangkang2318** [comment]
> It has to be in its own block, i.e. "display mode" of KaTeX. The `equation` environment can't be rendered inline.

 I check the code, and found that  the inline and block equation in the notion both are treated  as "text mode" of KaTeX when render them in the Web. But it should be that the inline equation is treated as “text mode”   and block equation is treated as "display mode".
So how do I  fix that?Thanks!

**Comment by y1zhou** [comment]
Yeah it seems the block equation is detected as `notion-equation-block` but KaTeX still thinks it's in text mode. Perhaps the issue lies in https://github.com/NotionX/react-notion-x/blob/master/packages/react-notion-x/src/third-party/equation.tsx?

**Comment by ShiinaRinne** [comment]
I have the same question, is this problem solved now?
